src/main/java/com/elytradev/concrete/block/ConcreteBlock.java

```diff
@@ -166,11 +155,45 @@ public Builder resistance(float resistance) {
             return this;
         }
 
-        public ConcreteBlock build(ConstructionBehaviour constructionBehaviour) {
+        public ConcreteBlock build() {
             checkNotNull(this.identifier, "An identifier is required to build a block!");
 
-            final ConcreteBlock block = constructionBehaviour.construct(this.identifier, this.material, this.drop, this.itemDropBehaviour,
-                    this.expDropBehaviour);
+            final ConcreteBlock block;
+            if (this.translucent) {
+                block = new ConcreteBlock(this.identifier, this.material, this.drop, this.itemDropBehaviour, this.expDropBehaviour) {
```

Comment:
One of these things is not like the others. All the other behaviors are handed to the constructor, and potentially retained by the block. Could we not pass a boolean opacity or an opacity behavior to the constructor instead of special-casing an anonymous class?